src/interactive/modules.ts

```diff
@@ -90,12 +91,14 @@ async function updateModuleForSelectionEvent(event: vscode.TextEditorSelectionCh
 async function updateModuleForEditor(editor: vscode.TextEditor) {
     const mod = await getModuleForEditor(editor)
 
-    let loaded = false
-    if (isConnectionActive()) {
-        loaded = await g_connection.sendRequest(requestTypeIsModuleLoaded, mod)
-    }
+    if (mod) {
+        let loaded = false
+        if (isConnectionActive()) {
+            loaded = await g_connection.sendRequest(requestTypeIsModuleLoaded, mod)
+        }
```

Comment:
I thought we were handling these situations with `try catch` now? I think we should, the `isConnectionActive()` pattern doesn't work in async code.

Reply:
True.